edb/pgsql/parser/ast_builder.py

```diff
@@ -156,6 +155,42 @@ def _unwrap(node: Node, name: str) -> Node:
     return node
 
 
+def _unwrap_boolean(n: Node) -> Node:
+    n = _unwrap(n, 'Boolean')
+    n = _unwrap(n, 'str')
+    n = _unwrap(n, 'boolval')
+    n = _unwrap(n, 'boolval')
+    if isinstance(n, dict) and len(n) == 0:
+        n = False
+    return n
+
+
+def _unwrap_int(n: Node) -> Node:
+    n = _unwrap(n, 'Integer')
+    n = _unwrap(n, 'str')
+    n = _unwrap(n, 'ival')
+    n = _unwrap(n, 'ival')
+    if isinstance(n, dict) and len(n) == 0:
+        n = 0
+    return n
+
+
+def _unwrap_float(n: Node) -> Node:
+    n = _unwrap(n, 'Float')
+    n = _unwrap(n, 'str')
+    n = _unwrap(n, 'fval')
+    n = _unwrap(n, 'fval')
+    return n
+
+
+def _unwrap_string(n: Node) -> Node:
+    n = _unwrap(n, 'String')
+    n = _unwrap(n, 'str')
+    n = _unwrap(n, 'sval')
+    n = _unwrap(n, 'sval')
```

Comment:
Double `sval`? What is libpg_query doing?

Reply:
I tested a few different versions of libpg and the behaviour changed slightly between each one. I kept the double unwrap just in case they decide to shift the behaviour back.